### DISCUSSION TITLE
Chromedriver device emulation

### DISCUSSION THREAD

**Discussion by rahul-raut** [webdriverio/webdriverio#7299] [Q&A] [answered]
I have used @wdio/devtools-service to emulate devices with CPU and network throttling. But it seems like if I don't use devtools service chromdriver is not able to emulate CPU and network throttling. Is that correct?

**Comment by christian-bromann** [comment, marked as answer]
> Is that correct?

Yes, the devtools service is currently only possible for local testing.

**Reply by rahul-raut** [reply]
What do you mean by local testing?

**Reply by christian-bromann** [reply]
By that I mean that the devtools service only works if your test run next to the browser (so on your local machine). Running the same tests on cloud provider such as Sauce Labs is not yet supported.

**Reply by rahul-raut** [reply]
Thank for clarification and your help !
